{"uuid":"b0f0963c-3b7c-4228-b6e3-065d5a4ea4e5","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["3e1c55a5-ff6a-47ca-9ace-b8d1a59ddbaf"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687658818212,"stop":1687658818213},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687658818557,"stop":1687658818557},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687658818780,"stop":1687658818780},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687658818915,"stop":1687658818915},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687658818932,"stop":1687658818933}],"start":1687658818046,"stop":1687658818933}